build/setup-oc.sh

```diff
@@ -8,6 +8,8 @@ if [ "$(uname -m)" = "s390x" ]; then
         url="http://mirror.openshift.com/pub/openshift-v4/s390x/clients/ocp/latest/openshift-client-linux.tar.gz"
 elif [ "$(uname -m)" = "x86_64" ];  then
         url="https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz"
+elif [ "$(uname -m)" = "aarch64" ];  then
+        url="https://mirror.openshift.com/pub/openshift-v4/aarch64/clients/ocp/latest/openshift-client-linux.tar.gz"
```

Comment:
Can we make this more generic? Something like 
` url="https://mirror.openshift.com/pub/openshift-v4/$(uname -m)/clients/ocp/latest/openshift-client-linux.tar.gz"`

Reply:
Good point - I'll admit - I was concerned by the odd specificity of the other lines, so I mirrored them - let's go ahead and make it the generic line that should cover all cases